static/js/services/vulnerability-service.js

```diff
@@ -127,7 +127,9 @@ angular.module('quay').factory('VulnerabilityService', ['Config', 'ApiService',
           'introducedInVersion': feature.Version,
 
           'imageId': addedByImageId,
-          'imageCommand': ImageMetadataService.getImageCommand(image, addedByImageId),
+          'imageCommand': addedByImageId.startsWith('sha256:')
```

Comment:
This will fail if/when we move to sha512. Instead, search for `:` in the name to check for a digest (since image IDs are hex only)

Reply:
Done